### ISSUE TITLE
ValueType convert to u8 start from 1;unknow start from 100

### ISSUE THREAD

**Issue by bartliu827** [cnosdb/cnosdb#451]

**Comment by azhsmesos** [comment]
Please assigned it to me, I want to contribute to cnosdb.

**Comment by bartliu827** [comment]
> Please assigned it to me, I want to contribute to cnosdb.

thanks！

**Comment by yinheli** [comment]
Hi, I've noticed that ValueType has been refactored and it appears to be different from when this issue was initially opened. 
It now corresponds to PhysicalDType in tskv. It seems that if i removing the implementation of `From<u8>`, it doesn't affect the compilation. 
So can we consider closing this issue?

**Comment by roseboy-liu** [comment]
> Hi, I've noticed that ValueType has been refactored and it appears to be different from when this issue was initially opened. It now corresponds to PhysicalDType in tskv. It seems that if i removing the implementation of `From<u8>`, it doesn't affect the compilation. So can we consider closing this issue?

No, ValueType corresponds to PhysicalDType in tskv. but it still remains unsolved

**Comment by yinheli** [comment]
I mean after comment the code, `make build` still works fine.

![image](https://github.com/cnosdb/cnosdb/assets/235094/a31c2b34-5c14-4458-97dd-30b6fcce0b1a)

**Comment by roseboy-liu** [comment]
> PhysicalDType

Got it, would you like to pull a request to remove the unused code and linked this issue.

**Comment by yinheli** [comment]
Sure.
